test/test-errors.js

```diff
@@ -200,3 +200,16 @@ test('trailing space should not influence the result', function(t) {
   t.equal(failures[0].type, 'string');
   t.end();
 });
+
+test('method name displayed on abstract function failure', function(t) {
+  var g = ohm.ohmGrammar;
+  var seq = g.ruleBodies.Grammar;
+  try {
```

Comment:
FYI, you can test this more succinctly like this:

    t.throws(function() { seq.toFailure(); }, /this method(toFailure) is abstract!/);